x-pack/plugins/security_solution/public/management/hooks/use_get_actions_state.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { UseQueryResult } from '@tanstack/react-query';
+import { useQuery } from '@tanstack/react-query';
+import type { IHttpFetchError } from '@kbn/core-http-browser';
+import { ACTION_STATE_ROUTE } from '../../../common/endpoint/constants';
+import { useHttp } from '../../common/lib/kibana';
+
+interface GetActionsStateResponse {
+  canEncrypt?: boolean;
+}
+/**
+ * Get info for actions state
+ */
+export const useGetActionsState = (): UseQueryResult<GetActionsStateResponse, IHttpFetchError> => {
```

Comment:
nit: Consider renaming this to `useGetActionState` for naming consistency, since the route const name is also singular.

Reply:
https://github.com/elastic/kibana/pull/154060/commits/01e4e262a5f771e443f2a8bc4085fff3ba3ecdc3